pkg/resmap/resmap.go

```diff
@@ -92,10 +92,18 @@ type ResMap interface {
 	// who's CurId is matched by the argument.
 	GetMatchingResourcesByCurrentId(matches IdMatcher) []*resource.Resource
 
+	// GetMatchingResourcesByCurrentIdAndCtx returns the resources
+	// who's CurId is matched by the argument.
+	GetMatchingResourcesByCurrentIdAndCtx(matches IdMatcher, rctxm resource.ResCtxMatcher) []*resource.Resource
+
 	// GetMatchingResourcesByOriginalId returns the resources
 	// who's OriginalId is matched by the argument.
 	GetMatchingResourcesByOriginalId(matches IdMatcher) []*resource.Resource
 
+	// GetMatchingResourcesByOriginalAndCtx returns the resources
+	// who's OriginalId is matched by the argument.
```

Comment:
ditto

Reply:
@Liujingfang1 Done. No method added to ResMap